clustering.Rmd

```diff
@@ -641,12 +777,13 @@ elbow_plot <- ggplot(clustering_statistics, aes(x = k, y = tot.withinss)) +
 elbow_plot
 ```
 
-
+What value should you choose for `nstart`? The answer is it depends.
```

Comment:
This paragraph should just be a couple sentences at the end of the last paragraph

Reply:
done